src/schema/metadata/PharmaceuticalDoseTime.yaml

```
@@ -0,0 +1,17 @@
+---
+name: PharmaceuticalDoseTime
+description: |
+  Time of administration of pharmaceutical dose, relative to time zero
+  (please see below).
+  For an infusion, this should be a vector with two elements specifying the
+  start and end of the infusion period. For more complex dose regimens,
+  the regimen description should be complete enough to enable unambiguous
+  interpretation of the DoseTime vector.
+  Unit format of the specified pharmaceutical dose time should be seconds.
```

Just out of curiosity: What is the `DoseTime` vector? I didn't find a field for this. Do they mean `PharmaceuticalDoseTime` ? If yes, we should change that.

```suggestion
  interpretation of the `DoseTime` vector.
  Unit format of the specified pharmaceutical dose time SHOULD be seconds.
```

Also, I feel the "SHOULD" in this sentence might as well have been a MUST 🤷

---

I think that it must be. `PharmaceuticalDoseTime` is the only "DoseTime" I can find in the spec.

---

Should we open a new issue about the `PharmaceuticalDoseTime` definition or just ping PET folks here?

---

Let's ping @mnoergaard and if it doesn't work we open an issue and ping more :)

---

Yes, you are absolutely right. This should be changed to the `PharmaceuticalDoseTime`.

---

Thanks!